synapse/federation/transport/server.py

```diff
@@ -1124,6 +1125,24 @@ def on_POST(self, origin, content, query):
         defer.returnValue((200, resp))
 
 
+class FederationGroupsJoinableServlet(BaseFederationServlet):
+    """Sets whether a group is joinable without an invite or knock
+    """
+    PATH = "/groups/(?P<group_id>[^/]*)/joinable$"
+
+    @defer.inlineCallbacks
+    def on_POST(self, origin, content, query, group_id):
```

Comment:
Should this be PUT or POST? The client API is a PUT.

Reply:
Erik said it should have the same method as the CS API. @erikjohnston, can you confirm?